src/frontend/src/common/components/library/data_table/index.tsx

```diff
@@ -33,7 +33,7 @@ const ITEM_HEIGHT_PX = 60;
 
 const LOADING_STATE_ROW_COUNT = 10;
 
-const UNDEFINED_TEXT = "---";
+const UNDEFINED_TEXT = "-";
```

Comment:
Don't want to pile extra work on you, but would you mind taking a look at the specialty `sequencingDate` renderer (`src/frontend/src/views/Data/cellRenderers.tsx`, line 118 on my local)? I previously made that to A) handle the weirdness of missing date being sent as `"N/A"` and B) to use a (at the time) special no-value string of `"-"`. But I think all that is pointless now and it can just become a `defaultCellRenderer`?

(B) is definitely handled if we're generally converting to a false-y value being shown with `-`. I'm not sure about (A) though -- is the backend now sending missing dates as `null` instead of `"N/A"`? If so, then yeah, `sequencingDate` renderer is no longer special and should be culled. If you don't have the time to take care of it with this PR, that's fine, just let me know and I'll make a note to fix it in the next week. Thanks!